### ISSUE TITLE
Proxy Domains

### ISSUE THREAD

**Issue by Victor6767** [MikeLime-dev/greenlime#13]
Could you make more proxies available since most are blocked

**Comment by QuiteAFancyEmerald** [comment]
@Victor6767 Join the TN discord for more proxy sites. I think you already are in the discord?

**Comment by Victor6767** [comment]
@QuiteAFancyEmerald  yeah i am but i can't get on it as often as I would like to but thank you for your help

**Comment by QuiteAFancyEmerald** [comment]
Oh okay of course no problem.
